twitter-as.js

```diff
@@ -130,7 +128,7 @@ var cli = new AppService.Cli({
 
           //Fix rooms that are alias rooms
           // Criteria: canonical_alias is #_twitter_@*+:domain
-          if (type == "timeline" && entry.matrix.get("twitter_user") == undefined) {
+          if (type === "timeline" && entry.matrix.get("twitter_user") == null) {
```

Comment:
It probably would make more sense to have === undefined in this case, since that's what it returns.